tour/ranges-splits.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Ranges and Splits
+---
+
+A Range is a specified group of Keys. There are many different ways to create a Range.  Here are a few examples:
+```java
+new Range(Key startKey, Key endKey)  // Creates a range from startKey inclusive to endKey inclusive.
+new Range(CharSequence row)  // Creates a range that covers an entire row.
```

Comment:
Its odd that Jekyll renders this `Range` as red but not the one on the line before.

Reply:
I was going to merge the content I had and then let @mikewalch clean it up.  Maybe he can figure out why.